platforms/Windows/bld/bld.wxs

```diff
@@ -0,0 +1,351 @@
+      <ComponentGroupRef Id="SwiftDemangle" />
+      <ComponentGroupRef Id="SwiftFeatures" />
+      <ComponentGroupRef Id="SwiftMigrator" />
+
+      <!-- TODO(compnerd) can we use symbolic links to swift-frontend.exe instead? -->
+      <Component>
+        <File Source="$(var.TOOLCHAIN_ROOT)\usr\bin\swift-api-digester.exe" />
+      </Component>
+      <Component>
+        <File Source="$(var.TOOLCHAIN_ROOT)\usr\bin\swift-autolink-extract.exe" />
+      </Component>
+      <Component>
+        <File Source="$(var.TOOLCHAIN_ROOT)\usr\bin\swift-symbolgraph-extract.exe" />
+      </Component>
+      <Component>
+        <File Source="$(var.TOOLCHAIN_ROOT)\usr\bin\swift-demangle.exe" />
+      </Component>
+      <Component>
+        <File Source="$(var.TOOLCHAIN_ROOT)\usr\bin\swift-frontend.exe" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="argument_parser" Directory="_usr_bin">
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\ArgumentParser.dll" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="tools_support_core" Directory="_usr_bin">
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\TSCBasic.dll" />
+      </Component>
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\TSCUtility.dll" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="swift_driver" Directory="_usr_bin">
+      <!-- TODO(compnerd) can we use symbolic links to swift.exe instead? -->
+      <Component>
+        <File Name="swiftc.exe" Source="$(var.DEVTOOLS_ROOT)\usr\bin\swift-driver.exe" />
+      </Component>
+
+      <Component>
+        <File Name="swift.exe" Source="$(var.DEVTOOLS_ROOT)\usr\bin\swift-driver.exe" />
+      </Component>
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\swift-help.exe" />
+      </Component>
+
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\SwiftOptions.dll" />
+      </Component>
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\SwiftDriver.dll" />
+      </Component>
+      <Component>
+        <File Source="$(var.DEVTOOLS_ROOT)\usr\bin\SwiftDriverExecution.dll" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="EnvironmentVariables">
+      <Component Id="SystemEnvironmentVariables" Condition="ALLUSERS=1" Directory="INSTALLDIR" Guid="d01ea5b8-0f8a-4388-9b61-1186efddfc39">
+        <Environment Id="SystemPath" Action="set" Name="Path" Part="last" Permanent="no" System="yes" Value="[INSTALLDIR]Toolchains\$(var.ProductVersion)+Asserts\usr\bin" />
+      </Component>
+      <Component Id="UserEnvironmentVariables" Condition="NOT ALLUSERS=1" Directory="INSTALLDIR" Guid="ab52b870-23ee-42e8-9581-3fcbdfb9228c">
+        <Environment Id="UserPath" Action="set" Name="Path" Part="last" Permanent="no" System="no" Value="[INSTALLDIR]Toolchains\$(var.ProductVersion)+Asserts\usr\bin" />
+      </Component>
```

Comment:
Would be good to know if we should preserve this - this is meant to support the dual-scope installers.